protocol/communities_messenger_test.go

```diff
@@ -2018,6 +2018,61 @@ func (s *MessengerCommunitiesSuite) TestShareCommunity() {
 
 	inviteMessage := "invite to community testing message"
 
+	// Create an community
+	response, err := s.bob.CreateCommunity(description, true)
+	s.Require().NoError(err)
+	s.Require().NotNil(response)
+	s.Require().Len(response.Communities(), 1)
+
+	community := response.Communities()[0]
+
+	response, err = s.bob.ShareCommunity(
+		&requests.ShareCommunity{
+			CommunityID:   community.ID(),
+			Users:         []types.HexBytes{common.PubkeyToHexBytes(&s.alice.identity.PublicKey)},
+			InviteMessage: inviteMessage,
+		},
+	)
+	s.Require().NoError(err)
+	s.Require().NotNil(response)
+	s.Require().Len(response.Messages(), 1)
+
+	// Add bob to contacts so it does not go on activity center
+	bobPk := common.PubkeyToHex(&s.bob.identity.PublicKey)
+	request := &requests.AddContact{ID: bobPk}
+	_, err = s.alice.AddContact(context.Background(), request)
+	s.Require().NoError(err)
+
+	// Pull message and make sure org is received
+	err = tt.RetryWithBackOff(func() error {
```

Comment:
`WaitOnMessengerResponse` should be used instead, it is superior to `RetryWithBackOff`.